gwas.yml

```diff
@@ -0,0 +1,8 @@
+name: gwas
+channels:
+  - conda-forge
+  - bioconda
```

Comment:
```suggestion
```
looks like we don't need bioconda in this case, since none of the packages come from it